app/lib/widgets/action_selection.dart

```diff
@@ -138,4 +153,14 @@ class _ActionSelectionState extends State<ActionSelection> {
                 resourceIds: resourceIds)));
     widget.refreshPatient(patient);
   }
+
+  void movePatient(Location moveTo) async {
+    await Navigator.push(
+        context,
+        MaterialPageRoute(
+            builder: (context) =>
+                MoveScreen(patient: widget.patient, moveTo: moveTo)));
+
+    widget.refreshPatient(null);
```

Comment:
Use response of request here, analog to performAction above; also see other comment

Reply:
done, see above